charms/worker/k8s/lib/charms/k8s/v0/k8sd_api_manager.py

```diff
@@ -598,6 +599,66 @@ class GetKubeConfigResponse(BaseRequestModel):
     metadata: KubeConfigMetadata
 
 
+class RefreshCertificatesPlanMetadata(BaseModel, allow_population_by_field_name=True):
+    """Metadata for the certificates plan response.
+
+    Attributes:
+        seed (int): The seed for the new certificates.
+        certificate_signing_requests (Optional[list[str]]): List of names
+        of the CertificateSigningRequests that need to be signed externally (for worker nodes).
+    """
+
+    seed: int
+    certificate_signing_requests: Optional[list[str]] = Field(
+        default=None, alias="certificate-signing-requests"
+    )
+
+
+class RefreshCertificatesPlanResponse(BaseRequestModel):
+    """Response model for the refresh certificates plan.
+
+    Attributes:
+        metadata (RefreshCertificatesPlanMetadata): Metadata for the certificates plan response.
+    """
+
+    metadata: RefreshCertificatesPlanMetadata
+
+
+class RefreshCertificatesRunRequest(BaseModel, allow_population_by_field_name=True):
+    """Request model for running the refresh certificates run.
+
+    Attributes:
+        seed (int): The seed for the new certificates from plan response.
+        expiration_seconds (int): The duration of the new certificates.
+        extra_sans (list[str]): List of extra sans for the new certificates.
+    """
+
+    seed: int
```

Comment:
Found it -- the response json has "seconds" here. 

```suggestion
    seed: int = Field(default=None, alias="seconds")
```